.github/workflows/releaseinfo.yml

```diff
@@ -0,0 +1,26 @@
+name: write release information to file
+
+on: workflow_dispatch
```

Comment:
Does it need to be a separate workflow? Why not make it as part of "publish" workflow?

Reply:
I think it's best to keep the workflows simple.
It's already getting very complicated.

Reply:
We also need to write `index.yaml` for helm charts, so this can be repurposed for that even.

Reply:
Okay. Who will call this workflow?

Reply:
The publish workflow will call this.

Reply:
Okay cool.

Reply:
also keeping it separate means we can trigger it manually if we want.